### PR TITLE
Merging new scopes and manifest features

### PR DESCRIPTION
**Pull request by Valentinolucas**
## Summary
- Add scopes by default
- Manifest spec guid

### CODE REVIEW

**Comment by Valentinolucas** [comment]
try deploying hello world pointing to master branch and it fails... will reopen when fixed

**Review by strutt — APPROVED** [review]

File: ansible/Makefile [hunk 1]
```diff
@@ -37,10 +40,10 @@ create-build-env-vars: guard-build_label guard-out_dir
 deploy-ecs-proxies: guard-account guard-build_label guard-service_id guard-APIGEE_ENVIRONMENT guard-PROXY_VARS_FILE
 	@poetry run ansible-playbook -i local deploy-ecs-proxies.yml
 
-deploy-apigee-proxy: guard-SERVICE_NAME guard-SERVICE_BASE_PATH guard-APIGEE_ENVIRONMENT guard-APIGEE_ORGANIZATION guard-APIGEE_ACCESS_TOKEN guard-PROXY_DIR guard-PING
+deploy-apigee-proxy: guard-FULLY_QUALIFIED_SERVICE_NAME guard-SERVICE_BASE_PATH guard-APIGEE_ENVIRONMENT guard-APIGEE_ORGANIZATION guard-APIGEE_ACCESS_TOKEN guard-PROXY_DIR guard-PING
 	@poetry run ansible-playbook -i local deploy-apigee-proxy.yml
 
-deploy-apigee-product-and-spec: guard-SERVICE_NAME guard-APIGEE_ENVIRONMENT guard-APIGEE_ORGANIZATION guard-APIGEE_ACCESS_TOKEN guard-FRIENDLY_NAME guard-PRODUCT_DISPLAY_NAME guard-PRODUCT_DESCRIPTION guard-VISIBLE guard-REQUIRE_CALLBACK_URL
+deploy-apigee-product-and-spec: guard-FULLY_QUALIFIED_SERVICE_NAME guard-APIGEE_ENVIRONMENT guard-APIGEE_ORGANIZATION guard-APIGEE_ACCESS_TOKEN guard-FRIENDLY_NAME guard-PRODUCT_DISPLAY_NAME guard-PRODUCT_DESCRIPTION guard-VISIBLE guard-REQUIRE_CALLBACK_URL
```
Comment (strutt) [inline ansible/Makefile:46]: Don't you still need SERVICE_NAME to get the scopes correct?

File: ansible/roles/deploy-apigee-product-and-spec/tasks/main.yml [hunk 2]
```diff
@@ -129,7 +129,7 @@
           quotaTimeUnit: minute
           environments:
             - "{{ APIGEE_ENVIRONMENT }}"
-          scopes: ['urn:nhsd:apim:app:jwks', 'urn:nhsd:apim:user:aal3' ]
+          scopes: ['urn:nhsd:apim:app:level3:{{ SERVICE_NAME }}', 'urn:nhsd:apim:user-nhs-id:aal3:{{ SERVICE_NAME }}' ]
```
Comment (strutt) [inline ansible/roles/deploy-apigee-product-and-spec/tasks/main.yml:132]: Here you use SERVICE_NAME but you don't pull it in in the pre-tasks?

Comment (strutt) [reply ansible/roles/deploy-apigee-product-and-spec/tasks/main.yml:132]: My mistake, yes you do!

**Review by Sourceless — APPROVED** [review]
Looks fine to me

**Pull request merged**